app/src/main/java/com/example/ubclaunchpad/launchnote/scan/scanCameraActivity.kt

```diff
@@ -0,0 +1,201 @@
+package com.example.ubclaunchpad.launchnote.scan
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.graphics.SurfaceTexture
+import android.hardware.camera2.*
+import android.os.Build
+import android.os.Bundle
+import android.support.design.widget.Snackbar
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import com.example.ubclaunchpad.launchnote.R
+
+import kotlinx.android.synthetic.main.activity_scan_camera.*
+import android.view.TextureView
+import android.util.Size
+import android.hardware.camera2.params.StreamConfigurationMap
+import android.support.v4.app.ActivityCompat
+import android.support.v4.content.ContextCompat
+import android.hardware.camera2.CameraAccessException
+import android.hardware.camera2.CameraDevice
+import android.opengl.ETC1.getHeight
+import android.opengl.ETC1.getWidth
+import android.view.Surface
+import android.hardware.camera2.CameraCaptureSession
+import android.os.HandlerThread
+import android.hardware.camera2.CameraMetadata
+import android.hardware.camera2.CaptureRequest
+import android.os.Handler
+import android.view.Window
+import android.view.WindowManager
+import com.example.ubclaunchpad.launchnote.photoBrowser.PhotoBrowserActivity
+
+
+class ScanCameraActivity : AppCompatActivity() {
```

Comment:
file name doesnt start with upper caps letter

Reply:
not sure why, it starts with upper case letter in my computer